packages/fast-components-react-msft/webpack.config.js

```diff
@@ -41,6 +42,10 @@ module.exports = {
             onBuildStart: [
                 `npm run convert:readme`
             ]
+        }),
+        new BundleAnalyzerPlugin({
```

Comment:
can we add this to the base package as well?

Reply:
yes we can.